scipy/special/_basic.py

```diff
@@ -14,7 +14,8 @@
 from ._ufuncs import (mathieu_a, mathieu_b, iv, jv, gamma,
                       psi, hankel1, hankel2, yv, kv, poch, binom,
                       _stirling2_inexact)
-from ._gufuncs import _lpn, _lpmn, _clpmn, _lqn, _lqmn, _rctj, _rcty
+from ._gufuncs import (_lpn, _lpmn, _clpmn, _lqn, _lqmn, _rctj, _rcty,
+                       _sph_harm_all as __sph_harm_all)
```

Comment:
How about `_sph_harm_all as _sph_harm_all_gufunc` to avoid these double leading underscores?

Reply:
Sure.